.github/workflows/publish.yml

```
@@ -0,0 +1,53 @@
+name: "Publish NPM package to repository"
+
+on: 
+  workflow_dispatch:
+    inputs:
+      tag_version:
+        description: 'The tag_version, e.g. vx.x.x'
+        default: 'vx.x.x'
+        required: true
+        type: 'string'
+      node_version:
+        description: 'The node_version to build with.'
+        default: '22.x'
```

Same as the create PR, shouldn't this be:

```suggestion
        default: '22'
```